src/test/java/unit/tests/testng/JsonParserTest.java

```diff
@@ -0,0 +1,71 @@
+package unit.tests.testng;
+
+import com.google.gson.Gson;
+import org.testng.annotations.*;
+import parser.*;
```

Comment:
Good practice is to avoid asterisks in imports. IDEA has settings for it, so you can setup it to prevent auto-collapse